### DISCUSSION TITLE
Git.custom_environment not working for Repo.clone_from in docker

### DISCUSSION THREAD

**Discussion by lileih** [gitpython-developers/GitPython#1166] [Q&A] [answered]
Dear Sirs/Madams,

I have problem with using Git.custom_environment and Repo.clone_from methods.
Running in a non-docker environment is successful and it fails to run under docker enviroment
mycode
  Repo.clone_from(sourcecode, localpath, branch=branch, env=dict(GIT_SSH_COMMAND=git_ssh_cmd)) 

it's my fault or problem is the code?

Thank you.

**Comment by Byron** [comment, marked as answer]
Please direct your questions to stackoverflow, providing the GitPython tag. This will help you formulate a question in such a way that people have a chance to help.

Thanks for your understanding.
